tests/filecheck/dialects/mod_arith/mod_arith.mlir

```diff
@@ -0,0 +1,10 @@
+// RUN: XDSL_ROUNDTRIP
+
+"builtin.module"() ({
+  %lhsi1, %rhsi1 = "test.op"() : () -> (i1, i1)
+
+  %add_res = "mod_arith.add"(%lhsi1, %rhsi1) : (i1, i1) -> i1
+
+  // CHECK: %add_res = "mod_arith.add"(%lhsi1, %rhsi1) : (i1, i1) -> i1
+
+}) : () -> ()
```

Comment:
I'd suggest a few changes:
```suggestion
// RUN: XDSL_ROUNDTRIP

%lhsi1, %rhsi1 = "test.op"() : () -> (i1, i1)

// CHECK: %add_res = "mod_arith.add"(%lhsi1, %rhsi1) : (i1, i1) -> i1
%add_res = "mod_arith.add"(%lhsi1, %rhsi1) : (i1, i1) -> i1
```
The builtin.module is implied, so we can skip that entirely.
The LLVM approach is usually to add check lines before the thing, as opposed to after. We're not consistent with this, but I've transitioned to this format now, and would recommend it also.